examples/step-13/step-13.cc

```diff
@@ -510,30 +505,30 @@ namespace Step13
     class Base
     {
     public:
-      Base (Triangulation<dim> &coarse_grid);
-      virtual ~Base ();
+      Base(Triangulation<dim> &coarse_grid);
+      virtual ~Base();
 
-      virtual void solve_problem () = 0;
-      virtual void postprocess (const Evaluation::EvaluationBase<dim> &postprocessor) const = 0;
-      virtual void refine_grid () = 0;
-      virtual unsigned int n_dofs () const = 0;
+      virtual void solve_problem() = 0;
+      virtual void postprocess(
+        const Evaluation::EvaluationBase<dim> &postprocessor) const = 0;
+      virtual void         refine_grid()                            = 0;
+      virtual unsigned int n_dofs() const                           = 0;
```

Comment:
The previous line 517 was way too long (the eyes have to travel a significant distance to spot what is going on).